test/cpp/jit/test_argument_spec.h

```diff
@@ -37,6 +39,20 @@ void testArgumentSpec() {
   auto& GF = at::CUDA(at::kFloat);
   auto& GD = at::CUDA(at::kDouble);
 
+  auto graph = std::make_shared<Graph>();
```

Comment:
Can you use `jit::compile(str)` to create the graph? We are trying to kill SymbolicVariable.